tools/testsys/Test.toml.example

```diff
@@ -11,6 +11,9 @@ repo = "default"
 # The registry containing alternate TestSys agent images
 testsys-images-registry = "public.ecr.aws/bottlerocket-test-system"
 
+# The tag that should be used with `testsys-images-registry` for image pulls
+testsys-images-registry = "latest"
```

Comment:
Do we push a "latest" tag for the TestSys images? I don't think we do for the other container images we publish under Bottlerocket, should we follow that rule here?

Reply:
We do not push to the latest tag. We only push the testsys version. Due to image cacheing in k8s clusters, latest should be used when debugging new images. This feature allows someone testing new agents to specify a tag and a registry instead of explicitly listing each agent.